### DISCUSSION TITLE
Is there a way to automatically save snapshots but only restore them upon user request?

### DISCUSSION THREAD

**Discussion by Tortenschachtel** [kangyu-california/PersistentWindows#353] [Q&A] [answered]
Hello. I am using the program for the 1 in 20 or so times when i wake up the computer from hibernation and Windows screws something up and my windows all get moved to the primary monitor. I would prefer not to have PersistentWindow restore my windows every time because that has side effects (the preview in the task bar does not match the actual window anymore). Is there a way i can let the program create snapshots automatically but only restore them manually? I am trying to "mimic" that by occasionally using "Capture windows to disk" and then "Restore windows from disk" if i need it, but i don't always remember to capture them.

**Comment by kangyu-california** [comment, marked as answer]
The side effect (of preview in taskbar does not match the actual window) does not seem to be related to PW auto restore.

Also please note that auto capture is paused during auto restore by design, skipping auto restore while keeping auto capture non-stopping would mess up the fundamental logics.
